main/src/mill/main/TokenReaders.scala

```diff
@@ -38,6 +42,9 @@ object TokenReaders {
   implicit def millEvaluatorTokenReader[T]: mainargs.TokensReader[Evaluator] =
     new mill.main.EvaluatorTokenReader[T]()
 
+  implicit def millAllEvaluatorsTokenReader[T]: mainargs.TokensReader[Seq[Evaluator]] =
+    new mill.main.AllEvaluatorsTokenReader[T]()
+
```

Comment:
Using a `Seq[Evaluator]` to express the wish to get all evaluators is a bit vague. A Seq can hold almost everything, also nothing. Should we introduce a dedicated type to hold all evaluators instead?

Reply:
Sure, I'll wrap it in `Evaluator.AllBootstrapEvaluators(value: Seq[Evaluator])`